uaaextras/webapp.py

```diff
@@ -383,6 +394,93 @@ def oauth_login():
     def index():
         return render_template('index.html')
 
+    @app.route('/signup', methods=['GET', 'POST'])
```

Comment:
We only pass the client creds to methods that need to auth as the client, not all methods.